src/commands/Admin/eval.ts

```diff
@@ -20,35 +22,74 @@ export default class extends Command {
 	}
 
 	async run(context: CommandContext, args: string[]): Promise<void> {
-		const clean = async (text: string) => {
-			if (typeof text === "string") return text.replace(/`/g, "`" + String.fromCharCode(8203)).replace(/@/g, "@" + String.fromCharCode(8203));
-			else return text;
-		};
+		let embed = new MessageEmbed()
+			.setFooter(
+				await this.client.bulbutils.translate("global_executed_by", context.guild?.id, {
+					user: context.author,
+				}),
+				<string>context.author.avatarURL({ dynamic: true }),
+			)
+			.setTimestamp();
 
-		try {
-			const code: string = args.join(" ");
-			let evaled = eval(code);
+		const start: number = new Date().getTime();
```

Comment:
```suggestion
		const start: number = Date.now();
```
`Date.now()` should be more efficient than constructing a Date instance